sdk/core/azure-core/src/test/java/com/azure/core/ConfigurationTests.java

```diff
@@ -95,7 +96,7 @@ public void notFoundConfigurationIsConvertedToNull() {
 
     @Test
     public void logLevelAndTracingDisabledUpdateInstantly() {
```

Comment:
This test case could be split into two test cases, because it tests 2 scenarios, AZURE_LOG_LEVEL and AZURE_TELEMETRY_DISABLED. Even if AZURE_LOG_LEVEL was set in the environment, AZURE_TELEMETRY_DISABLED might not be.

Since you change a system variable in line 99 and another instance below, you should reset the environment variables when this test completes (successfully or otherwise).